authors/apps/articles/serializers.py

```diff
@@ -86,3 +87,15 @@ def get_slug(self, obj):
 
     def get_description(self,obj):
        return obj.article.description
+
+
+class FavoriteSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Article
+        fields = [
+            'title',
+            'body',
+            
+        ]
```

Comment:
Consider adding a slug to the returned fields. This will allow a user to link back to the main article through their favorites.

Reply:
Okay, let me do that. Thanks.